src/Sarif/CodeGenHints.json

```diff
@@ -1075,6 +1095,26 @@
       }
     }
   ],
+  "TranslationMetadata.DownloadUri": [
```

Comment:
>DownloadUri [](http://example.com/codeflow?start=23&length=11)

Good changes, but hm. how'd you find them?

Reply:
not sure how we missed them.
I found them by test not eye.